RELEASES.md

```diff
@@ -0,0 +1,131 @@
+# 🚀 Releases
+
+This document outlines the Moby project’s release process, including cadence and versioning.
+
+## 📌 Versioning
+
+Releases of moby modules will be versioned using dotted triples, similar to
+[Semantic Version](http://semver.org/). For the purposes of this document, we
+will refer to the respective components of this triple as
+`<major>.<minor>.<patch>-[suffix[N]]`. The version number may have additional
+information, such as alpha, beta and release candidate qualifications along with
+a numeric identifier. Such releases will be considered "pre-releases".
+
+> [!NOTE]
```

Comment:
We can probably create a separate section about "relation to the docker/docker" module.

(or at least, most of this information is/becomes obsolete, and describes a different module)